pkg/logs/status/status.go

```diff
@@ -111,3 +119,17 @@ func toDictionary(c *config.LogsConfig) map[string]interface{} {
 	}
 	return dictionary
 }
+
+func generateOpenFilesLimitReachedWarning(c *config.LogsConfig, inputs []string) string {
+	if c.Type != config.FileType || !Warnings.RaisedWarnings[OpenFilesLimitReachedType] {
+		return ""
+	}
+	if count, ok := Warnings.Wildcards[c.Path]; ok {
+		if count == len(inputs) {
+			return fmt.Sprintf("This path tails %d files", count)
+		}
+		return fmt.Sprintf("This path tries to tail %d files but only %d files are tailed", count, len(inputs))
```

Comment:
General nit: I'd avoid `tries to` and `but only` in agent statuses, as those tend to be blurry words.
What about something like `%d files tailed out of %d files matching` that we could use for both cases?

Reply:
👍